sweagent/agent/models.py

```diff
@@ -690,16 +690,21 @@ def get_model(args: ModelArguments, commands: Optional[list[Command]] = None):
     if commands is None:
         commands = []
 
-    if args.model_name == "human":
-        return HumanModel(args, commands)
-    if args.model_name == "human_thought":
-        return HumanThoughtModel(args, commands)
-    if args.model_name == "replay":
-        return ReplayModel(args, commands)
-    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt"):
+    models = {
```

Comment:
honestly perhaps let's just bring that back to the original version (conflicting anyway). Shouldn't make too many changes while we don't have proper CI in place